src/ducks/recurrence/search.spec.js

```diff
@@ -143,7 +143,7 @@ describe('recurrence scenario with all operations which have a subcategory', ()
     automaticLabel: 'Mon Salaire'
   }
 
-  it('should match 01/06 salary with 2000 and salary in subcategory', () => {
+  it('should reattach salary of the month of June (2000)', () => {
```

Comment:
comme on est dans un describe `recurrence scenario with multiple amounts` on devrait peut-être plutôt parler de amount dans les `it` non ? Le `it` devrait être en rapport avec le `expect` je pense non ?

En fait ici on veut vérifier qu'avec une nouvelle transaction, mais avec un montant différent, ça la rattache bien à un bundle déjà existant ? Si c'est bien ça, on devrait retrouver ce postulat dans la description du `it`